layers/+tags/gtags/packages.el

```diff
@@ -11,13 +12,24 @@
 
 (setq gtags-packages
   '(
+    eldoc
     helm-gtags
     ggtags
     ))
 
 (defun gtags/init-ggtags ()
   (use-package ggtags
-    :defer t))
+    :defer t
+    :init
+    (progn
+      ;; modes that do not have a layer, add here.
+      (add-hook 'awk-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'shell-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'tcl-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'vhdl-mode-hook (lambda () (ggtags-mode 1)))
```

Comment:
Joe Hillenbrand <notifications@github.com> writes:
>> +      (add-hook 'vhdl-mode-hook (lambda () (ggtags-mode 1)))
>
> Should it also be a toggle?

What circumstance are you imaging one would wish to toggle gtags mode on
and off?

Thanks,
Chris.